src/Interpreter.ts

```diff
@@ -162,9 +169,18 @@ export default class Interpreter {
    * Interprets an image as a ballot, returning information about the votes cast.
    */
   public async interpretBallot(
+    imageData: ImageData,
+    { metadata }: { metadata?: BallotPageMetadata } = {}
+  ): Promise<Interpreted> {
+    const marked = await this.findMarks(imageData, metadata)
+    const ballot = this.interpretMarks(marked)
+    return { ...marked, ballot }
+  }
+
+  private async findMarks(
```

Comment:
I made this and `interpretMarks` private for now, but we may want them to be public later.